packages/vendor-core/lib/modules.js

```diff
@@ -91,4 +91,6 @@ module.exports = [
   'urijs',
   '@novnc/novnc/core/rfb',
   '@spice-project/spice-html5',
+  '@webcomponents/webcomponentsjs/webcomponents-bundle.js',
+  '@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js',
```

Comment:
I believe the `.js` is not required

Reply:
going to test it :)

Reply:
I believe those files are not get imported from core/plugins and they just need to be apart of the bundle?

If you would want to import them, you will need to use the same convention as here, e.g. use `.js` in your imports as well (in foreman/plugins).

Reply:
true the `.js` is not needed :+1: